packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.scss

```diff
@@ -26,7 +26,7 @@
         cursor: pointer;
         margin: 3.4rem 0.8rem 2.4rem 6.4rem;
         position: absolute;
-        left: 0;
+        left: -2.4rem;
```

Comment:
There's a margin of 6.4rem, can you try adjusting that instead of giving negative left property and please verify it with responsive width

Reply:
And I don't understand why there's a `position: absolute;` property just for a text

Reply:
I'm not sure why it was used here.
thank for the remark, will refactor it